tests/testthat/test-next_cursor.R

```diff
@@ -0,0 +1,22 @@
+test_that("max_id and since_id work in bit64", {
+  expect_equal(max_id("0"), "-1")
+  expect_equal(max_id(c("50", "100")), "49")
+  
+  expect_equal(since_id(c("50", "100")), "100")
+})
```

Comment:
Isn't the name a bit misleading? Wouldn't a better test name be max_id and since_id work in characters? Maybe there should be a different test for bit64 numbers.

Reply:
They work _in_ bit64 space with character vectors.